File: app/src/main/java/de/bitb/astroskop/ui/main/profiles/details/IProfileDetailView.java
package de.bitb.astroskop.ui.main.profiles.details;

import de.bitb.astroskop.model.Profile;
import de.bitb.astroskop.ui.base.IBaseView;

interface IProfileDetailView extends IBaseView {

    void setToolbarTitle(String title);

    void initProfile(Profile profile);

    void refreshView();
}

File: app/src/main/java/de/bitb/astroskop/ui/main/profiles/details/IProfileDetailView.kt
package de.bitb.astroskop.ui.main.profiles.details

import de.bitb.astroskop.model.Profile
import de.bitb.astroskop.ui.base.IBaseView

internal interface IProfileDetailView : IBaseView {

    fun setToolbarTitle(title: String)

    fun initProfile(profile: Profile)

    fun refreshView()
}
